docs/how-to-guides/basic_image.rst

```diff
@@ -0,0 +1,99 @@
+.. _howto_buld_basic_image:
+
+===================
+Build a basic image
+===================
+
+These instructions describe how to build a ``classic`` server 24.04 AMD64 Ubuntu image
+with Imagecraft.
+
+Prerequisites
+-------------
+
+- AMD64 machine with Ubuntu 24.04
+- ``snapd`` installed
+- 15GiB or more disk space to process the build and hold the resulting image
+
+.. note:: Following instructions build an **AMD64** image on an **AMD64** machine.
+          Building on another architecture would need several modifications not
+          described on this page.
+
+Install Imagecraft
+~~~~~~~~~~~~~~~~~~
+
+Imagecraft is only available as a snap in the ``latest/edge`` channel
+`from the Snapstore <https://snapcraft.io/imagecraft>`_. Install it with:
+
+.. code-block::
+
+    sudo snap install --classic --edge imagecraft
+
+Verify that ``imagecraft`` is properly installed:
+
+.. code-block::
+
+    imagecraft
+
+.. important:: ``imagecraft`` requires **elevated permissions**. Run it with **root**
+               privileges or using ``sudo`` if you plan on using the executing system
+               as the build system ("destructive mode").
```

Comment:
I'm having a hard time understanding the nuances here. I think we should make some changes to this callout.

- If Imagecraft _requires_ elevated permissions, why do we say "if you plan on using..."? The second phrase makes it sound optional.
- Since we're warning about destructive mode here, what are the potential side effects? It sounds dangerous – can we describe how?
- Maybe this callout should be a caution, not a warning? We use cautions for when a user action could damage data or hardware.

Reply:
I replaced this with a caution callout warning about the risk of using the tool. For now the rest of the how-to only mention running the tool with `sudo` to pack an image so I will revisit that when I am sure the tool can work well in lxd/multipass.

Reply:
Looks good, thanks!